docs/data-ai/data/hot-data-store.md

```diff
@@ -0,0 +1,164 @@
+---
+linkTitle: "Speed up queries to recent data"
```

Comment:
this is a bit long for the side bar.

Reply:
Maybe?
```suggestion
linkTitle: "Optimize recend data queries"
```
Anything that stays in imperative mood but doesn't spread across two lines